Add 1 on 1 video calling/client.js

```diff
@@ -0,0 +1,152 @@
+import { CallClient, CallAgent, Renderer, LocalVideoStream} from "@azure/communication-calling";
+import { AzureCommunicationTokenCredential } from '@azure/communication-common';
+
+let call;
+let callAgent;
+const calleeInput = document.getElementById("callee-id-input");
+const callButton = document.getElementById("call-button");
+const hangUpButton = document.getElementById("hang-up-button");
+const stopVideoButton = document.getElementById("stopVideo");
```

Comment:
`stopVideo` and `startVideo` also follow the convention above? eg. `stop-video`, `start-video`?

Reply:
Fixed.